### PR TITLE
[fix] Separate official language data, add missing sidebar items

### LINKED ISSUE #371: As an employee, I need to be able to hide my OL information and security independently/separately
**Is your feature request related to a problem? Please describe.**

On profile page, the employee summary card doesn't allow employees to just hide their OL information or security clearance information separately. Also, this information should be set to hidden by default.

**Describe the solution you'd like**
Employees should be able to hide OL information or clearance information.
Also, the toggles for  OL information and clearance information should be set to hidden by default.
It may be necessary to create a separate card for OL results and separate card for the Security 
  

**Additional context**
image

### LINKED ISSUE #372: Side navigation bar on employee profile page missing headings
**Is your feature request related to a problem? Please describe.**
The side nav is missing several subheadings from the employee profile cards. (eg. substantive, A/Classification, OL, Recent talent mgt results

**Describe the solution you'd like**
The side nave should list all Headings, and subheadings in the appropriae order.

**Additional context**
Check with Kate to ensure it also passes accessibility   
Coimplet this task after issue  #371 

image

### PR DESCRIPTION
**Pull request by TrevorBivi**
- Separates official language information tab into it's own card
- merges substantive and acting tabs from original card into a new card without tabs
- Fixes missing sidebar items

_wide layout without acting classification_
actinglessWideView

_wide layout with acting classification_
wideView

_narrow layout with acting classification_
narrowView

### CODE REVIEW

**Comment by MenaMachado** [comment]
Hi Trevor,  Ali and I spoke, for employee summary card, go back to using tabbing for the substantive and acting. 

Also rename: Employee Summary to Substantive in the Heading. 

And just below create a separate card with toggle switch for Security.

File: services/frontend-v3/src/i18n/fr_CA.json [hunk 1]
```diff
@@ -272,10 +272,10 @@
   "profile.location": "Emplacement",
   "profile.manager": "Gestionnaire",
   "profile.official.language": "Langue officielle",
-  "profile.oral": "Oral",
-  "profile.secondary.oral.date": "Date de l'évaluation",
+  "profile.oral": "niveau oral",
+  "profile.secondary.oral.date": "Date d'expiration",
   "profile.secondary.oral.proficiency": "Compétence à l'oral",
-  "profile.reading": "Lecture",
+  "profile.reading": "niveau de lecture",
```
Comment (BenJeau) [inline services/frontend-v3/src/i18n/fr_CA.json:278]: Would capitalize first letter (to stay consistent with "Niveau d'écriture")
```suggestion
  "profile.reading": "Niveau de lecture",
```

File: services/frontend-v3/src/i18n/fr_CA.json [hunk 2]
```diff
@@ -272,10 +272,10 @@
   "profile.location": "Emplacement",
   "profile.manager": "Gestionnaire",
   "profile.official.language": "Langue officielle",
-  "profile.oral": "Oral",
-  "profile.secondary.oral.date": "Date de l'évaluation",
+  "profile.oral": "niveau oral",
```
Comment (BenJeau) [inline services/frontend-v3/src/i18n/fr_CA.json:275]: Same here
```suggestion
  "profile.oral": "Niveau oral",
```

File: services/backend/src/database/migrations/20191031145350-create-user-and-profile.js [hunk 3]
```diff
@@ -140,6 +140,7 @@ module.exports = {
               careerInterests: true,
               mentorshipSkills: true,
               exFeeder: true,
+              security: false,
```
Comment (BenJeau) [inline services/backend/src/database/migrations/20191031145350-create-user-and-profile.js:143]: Would need to add a new field to the `schema.prisma` file in the database, then create a new migrate based on the prisma cli or by doing `yarn migrate:create`, then apply the migration with `yarn migrate`, and probably also update the seeds file under database > seed > fake profile/user data

**Review by BenJeau — COMMENTED** [review]
I'll test it once the changes are brought to the backend and there's no merge conflicts, but looks good otherwise!

**Comment by MenaMachado** [comment]
@TrevorBivi, can you remove "grade", it can say "reading", "writing", "oral"

**Comment by MenaMachado** [comment]
Trevor, For the employee summary card, are you able to get substantive, classification, and security in one row? Same with the Acting tab. 

If you can get it in a row can you then place the language info box just below it with a row for the first language and a separate row for each grade, valid/expired toggle switch and expiry date? 

You may need to stretch the length of contact card a bit.  We will be adding a space to allow for a position title Below the name.

**Pull request closed without merging**